exporter/awsxrayexporter/internal/translator/cause.go

```diff
@@ -160,6 +160,8 @@ func parseException(exceptionType string, message string, stacktrace string, lan
 		exceptions = fillJavaScriptStacktrace(stacktrace, exceptions)
 	case "dotnet":
 		exceptions = fillDotnetStacktrace(stacktrace, exceptions)
+	case "php":
+		exceptions = fillJavaStacktrace(stacktrace, exceptions)
```

Comment:
Should this be php stack trace?

Reply:
The PHP stacktrace is in the form of a java stack trace so we use the fillJavaStackTrace() function